translate/convert/web2py2po.py

```diff
@@ -24,9 +24,12 @@
 for examples and usage instructions.
 """
 
-from translate.storage import po
 import sys
 
+import six
```

Comment:
`sys` import above is no longer used.

Also according to our styleguide the imports should be in this order: http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#imports `six` counts as a "Third party library import".